FINAL_COUNTDOWN.md

```diff
@@ -0,0 +1,74 @@
+
+### Component Helper
+- [ ] Implement `actions` on the `component` helper: [Failing test 6](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L402)
+
+### Class Helper
+- [ ] Implement `recompute` hook: [Failing test 1](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/custom-helper-test.js#L44) [Failing test 2](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/custom-helper-test.js)
+- [ ] Implement teardown: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/custom-helper-test.js#L325)
+
+# Failing Tests
+
+### Curly Component
+
+- [ ] Ensure `elementId` cannot change: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L48)
+- [ ] Ensure `classBinding` without a condition applies correctly: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L176)
+- [ ] Ensure falsy values do not become class names: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L188)
+- [ ] Ensure `attributeBindings` cannot be set: [Failing Test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L453)
```

Comment:
Can someone investigate where that assertion is coming from? We probably just need to add it back to our ["slimmed down" component class](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/lib/ember-views/component.js).